### DISCUSSION TITLE
Can't upgrade Swift package dependency to 0.34.0

### DISCUSSION THREAD

**Discussion by Pigpocket** [pointfreeco/swift-composable-architecture#1059] [Q&A]
**Describe the bug**
My project currently is currently depending on version `0.29.0`, but seems to not allow me to depend on the latest version, `0.34.0 `. I get the following error: 

`Dependencies could not be resolved because root depends on 'swift-composable-architecture' 0.34.0 and root depends on 'swift-composable-architecture' 0.29.0.`

This prevents the majority of my other dependencies from fetching as well.

**To Reproduce**
I've tried the standard methods of Updating to Latest Package Versions, Resolve Package Versions and Reset Swift Package caches, as well as deleting Derived Data, cleaning the project, closing and re-opening both Xcode and the project. This feels Xcode-y, but is only happening for me with this repo. To reproduce this issue, I change the version number string in my `Package.swift` file, like below. I have also tried it without the `name:` parameter (that's how I originally had it). 

```swift
.package(name: "swift-composable-architecture", url: "https://github.com/pointfreeco/swift-composable-architecture", .exact("0.34.0")),
```

**Expected behavior**
I can set my TCA dependency to 0.34.0

**Screenshots**

**Environment**
  - swift-composable-architecture version 0.34.0
  - Xcode 13.2.1
  - Swift 5.5
  - OS: iOS 15

**Comment by mbrandonw** [comment]
Hi @Pigpocket, can you share a little bit more information about your set up? Either your Package.swift or a reproduction of the problem in a small project? The error makes it sound like somehow you have two dependencies on the library. 

I don't believe this is an error with the library so I am going to convert this issue to a discussion.

**Comment by Pigpocket** [comment]
It was not an error with the library. It was, alas, an error with me, as usual. Sadness becomes me. While I updated the version number in the `Package.swift` file, I didn't do it in the Package Dependencies section of the project. Dumb omission. Sorry for the confusion!

By the way, I was trying to use `.synchronize` in order to take advantage of `@FocusedState`. That isn't part of TCA library yet and I should just adopt the `View` extension at this point?

**Comment by mbrandonw** [comment]
No worries, glad you figured it out.

And yes, `synchronize` is not apart of the library because it doesn't really depend on anything in TCA. We recommend you copy-and-paste it as you need.
